html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="example.php" method="get" name="name1">
+    <div>
+      <label for ="address">Email</label><br>
+      <input type="text" name="email" id="address">
+    </div>
+    <div>
+      Why is coding cool?<br>
+      <select name="reason">
```

Comment:
Each input requires a label, and you should have matching for and id attributes as per the documentation.

Reply:
![image](https://user-images.githubusercontent.com/78944582/111019645-95a59080-8375-11eb-9dc5-03732e603710.png)